providers/gov/pubmedcentral/pmc/migrations/0001_initial.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-09-13 15:20
+from __future__ import unicode_literals
+
+from django.db import migrations
+import share.robot
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('share', '0001_initial'),
+        ('djcelery', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            code=share.robot.RobotUserMigration('gov.pubmedcentral.pmc'),
+        ),
+        migrations.RunPython(
+            code=share.robot.RobotOauthTokenMigration('gov.pubmedcentral.pmc'),
+        ),
+        migrations.RunPython(
```

Comment:
This should only have the `RobotScheduleMigration` the pubmed user already exists.